Enigmatry.Entry.Blueprint.Api.Tests/Features/UsersControllerFixture.TestUpdate.verified.txt

```diff
@@ -4,5 +4,8 @@
   FullName: some user,
   RoleId: Guid_2,
   CreatedOn: DateTimeOffset_1,
-  UpdatedOn: DateTimeOffset_1
+  UpdatedOn: DateTimeOffset_1,
+  UserStatusId: Inactive,
```

Comment:
UserStatusId: Inactive was achieved with Enigmatry.Entry.SmartEnums.VerifyTestsJson.ConvertersExtensions.EntryAddSmartEnumJsonConverters.

If this was not registered the Verify would contain both the Id and the name in the UserStatusId which makes reading Verify files a bit more dificult and confusing.